Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -402,6 +402,16 @@ public final class MapboxMap: MapboxMapProtocol {
             forCamera: MapboxCoreMaps.CameraOptions(camera))
     }
 
+    /// Returns the unwrapped coordinate bounds to a given `CameraOptions`.
+    ///
+    /// This function is particularly useful, if the camera shows the antimeridian.
+    ///
+    /// - Parameter camera: The camera for which the coordinate bounds will be returned.
+    /// - Returns: `CoordinateBounds` for the given `CameraOptions`
```

Comment:
```suggestion
    /// Returns the unwrapped coordinate bounds to a given ``CameraOptions``.
    ///
    /// This function is particularly useful, if the camera shows the antimeridian.
    ///
    /// - Parameter camera: The camera for which the coordinate bounds will be returned.
    /// - Returns: `CoordinateBounds` for the given ``CameraOptions``.
```

For symbols that are part of mapbox-maps-ios, we can use double back ticks `\`\`` to add links to documentation. Links to symbols declared in GL Native are not yet supported.